src/graphics/mod.rs

```diff
@@ -980,20 +844,30 @@ pub trait Drawable {
 
 /// Applies `DrawParam` to `Rect`.
 pub fn transform_rect(rect: Rect, param: DrawParam) -> Rect {
```

Comment:
where is this `transform_rect()` actually used? because i have no idea if Matrix mode might actually be something we need to implement, if the lack of it might break something.

Reply:
It is only used in `SpriteBatch` (and then also `Text`) to figure out the dimensions of transformed sub-sprites.

Reply:
hmm, text case is actually something that would break 🤔 i'll think about it and ping you back with some idea.

Reply:
actually, one idea i have is when you use Matrix, it assumes you've already stored whole rectangle transformation in it, so if we always work with 1x1 unit size thing (because iirc you've told me that we always work with 1x1 quad), we could just calculate rect by multiplying each source rect corner position (0,0; 1,0; 1,1; 0,1) and by that we get that 1x1 quat corners locations but in world space, from that we can calculate rectangle out of it. it is only an idea, so it would require tests.